app/Models/Signup.php

```diff
@@ -22,7 +22,7 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'campaign_run_id',
+        'campaign_id', 'campaign_run_id', 'updated_at',
```

Comment:
@chloealee yeah stuff that we query by we want to be an index so it's speedier. I'm not sure that _everything_ that we query by needs to be an index, but it seems safe to me to do that for columns that `/activity` uses. @DFurnes might have a different opinion or more to say though!